src/components/ChatApp/MessageListItem/Feedback.react.js

```diff
@@ -109,22 +97,22 @@ class Feedback extends React.Component {
     const defaultFeedbackColor = theme === 'light' ? '#90a4ae' : '#7eaaaf';
 
     return (
-      <span>
+      <div>
         {message && message.authorName === 'SUSI' ? (
-          <span className="feedback" style={styles.feedbackContainer}>
+          <div className="feedbackContainer">
             <ThumbUp
               onClick={() => this.postSkillFeedback('positive')}
-              style={styles.feedbackButton}
-              color={feedback === 'positive' ? '#00ff7f' : defaultFeedbackColor}
+              style={{ height: '16px', cursor: 'pointer' }}
```

Comment:
Use class for this as well

Reply:
@akshatnitd height was being set by MaterialUI. A class couldn't override it